staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+topologyPolicies:
```

Comment:
Do we really want to mix policies with declarative state?

Reply:
Here just a list of policies for resource management on the worker node. It's a list, not a single value since now it could be

Reply:
The on-node agent is the only owner here, it determines the node topology and populates the CR. In addition to inspecting the resource distribution across NUMA nodes, it detects how kubelet has been configured (e.g. Topology manager policy) by inspecting the kubelet config and captures that information in`topologyPolicies` field. This information exposed can be useful for the consumers of this API e.g. scheduler plugin making scheduling decisions using the CR. The configured Topology Manager policy influences the scheduler plugin decision and hence has been captured here.